website/content/docs/nia/api/status.mdx

```diff
@@ -6,28 +6,27 @@ description: >-
 ---
 # Status
 
-The `/status` endpoints share status-related information for tasks. This information is available for understanding the status of individual tasks and across tasks.
+The `/status` endpoints return status-related information for tasks. This endpoint returns a count of successful and failed task events that are recorded whenever tasks execute an automation. Currently, only the five most recent events are stored in Consul-Terraform-Sync (CTS). For more information on the hierarchy of status information and how it is collected, refer to [Status Information](/docs/nia/tasks#status-information).
 
-The health status value is determined by aggregating the success or failure of the event of a task detecting changes in Consul services and then updating network infrastructure. Currently, only the 5 most recent events are stored in Consul-Terraform-Sync (CTS). For more information on the hierarchy of status information and how it is collected, see [Status Information](/docs/nia/tasks#status-information).
+If CTS is configured [for high availability](/docs/nia/usage/run-ha), you can send a request to the [`/status/cluster` endpoint](#cluster-status) of any instance to get information about the cluster. Requests to the `status` endpoint return a 400 error, however, if high availability is enabled and the request is sent to a follower instance. The error message depends on what information the follower instance is able to obtain about the leader. Refer to [Error Messages](/docs/nia/usage/errors-ref) for more information.
```

Comment:
This paragraph is a bit confusing. You can send requests to `/status/cluster` regardless of it being a follower or a leader. This contradicts `Requests to the `status` endpoint return a 400 error, however, if high availability is enabled and the request is sent to a follower instance.`

Maybe something like this is clearer:

```suggestion
If CTS is configured [for high availability](/docs/nia/usage/run-ha), with exception to the [`/status/cluster` endpoint](#cluster-status), requests to the `status` endpoint on follower instances return a 400 error. The error message depends on what information the follower instance is able to obtain about the leader. Refer to [Error Messages](/docs/nia/usage/errors-ref) for more information. With high availability configured you can send requests to the `status/cluster` endpoint on any CTS cluster member instance to receive information about the entire cluster. 
```